core/src/main/scala/clue/package.scala

```diff
@@ -9,6 +9,8 @@ import cats.syntax.all._
 import org.typelevel.log4cats.Logger
 
 package object clue {
+  type FetchClient[F[_], S] = FetchClientWithPars[F, _, S]
```

Comment:
Can the `_` be `Unit` or something? Wildcard types are weird

Reply:
We actually want it to match any type. In other words, we want to ignore the type parameter if we are not going to do anything with it.